src/quantum/config.js

```diff
@@ -62,4 +65,89 @@ const CONFIG = {
 
 };
 
-export default CONFIG;
+
+const PLATFORMS = [
+  {
+    browser: 'Firefox',
+    bits: '32',
+    os: 'win',
+    label: 'Firefox',
+    frameworkId: 10,
+    platform: 'windows7-32',
+    option: 'pgo',
+    project: 'mozilla-central',
+  },
+  {
+    browser: 'Firefox',
+    bits: '64',
+    os: 'win',
+    label: 'Firefox',
+    frameworkId: 10,
+    platform: 'windows10-64',
+    option: 'pgo',
+    project: 'mozilla-central',
+  },
+  {
+    browser: 'Chrome',
+    bits: '32',
+    os: 'win',
+    label: 'Chrome',
+    frameworkId: 10,
+    platform: 'windows7-32-nightly',
+    option: 'opt',
+    project: 'mozilla-central',
+  },
+  {
+    browser: 'Chrome',
+    bits: '64',
+    os: 'win',
+    label: 'Chrome',
+    frameworkId: 10,
+    platform: 'windows10-64-nightly',
+    option: 'opt',
+    project: 'mozilla-central',
+  },
+];
+
+const SUITES = {
+  header:
+    ['browser', 'title', 'suite'],
+
+  data: [
+
+    ['Firefox', 'Tp6: Facebook', 'raptor-tp6-facebook-firefox'],
+    ['Firefox', 'Tp6: Amazon', 'raptor-tp6-amazon-firefox'],
+    ['Firefox', 'Tp6: YouTube', 'raptor-tp6-youtube-firefox'],
+    ['Firefox', 'Tp6: Google', 'raptor-tp6-google-firefox'],
+    ['Firefox', 'Tp6: imdb', 'raptor-tp6-imdb-firefox'],
+    ['Firefox', 'Tp6: imgur', 'raptor-tp6-imgur-firefox'],
+    ['Firefox', 'Tp6: wikia', 'raptor-tp6-wikia-firefox'],
+    ['Firefox', 'Tp6: bing', 'raptor-tp6-bing-firefox'],
+    ['Firefox', 'Tp6: yandex', 'raptor-tp6-yandex-firefox'],
+    ['Firefox', 'Tp6: apple', 'raptor-tp6-apple-firefox'],
+    ['Firefox', 'Tp6: microsoft', 'raptor-tp6-microsoft-firefox'],
+    ['Firefox', 'Tp6: reddit', 'raptor-tp6-reddit-firefox'],
+
+    ['Chrome', 'Tp6: Facebook', 'raptor-tp6-facebook-chrome'],
+    ['Chrome', 'Tp6: Amazon', 'raptor-tp6-amazon-chrome'],
+    ['Chrome', 'Tp6: Google', 'raptor-tp6-google-chrome'],
+    ['Chrome', 'Tp6: YouTube', 'raptor-tp6-youtube-chrome'],
+    ['Chrome', 'Tp6: imdb', 'raptor-tp6-imdb-chrome'],
+    ['Chrome', 'Tp6: imgur', 'raptor-tp6-imgur-chrome'],
+    ['Chrome', 'Tp6: wikia', 'raptor-tp6-wikia-chrome'],
+    ['Chrome', 'Tp6: bing', 'raptor-tp6-bing-chrome'],
+    ['Chrome', 'Tp6: yandex', 'raptor-tp6-yandex-chrome'],
+    ['Chrome', 'Tp6: apple', 'raptor-tp6-apple-chrome'],
```

Comment:
Could you please uppercase the name of all the sites? (e.g. apple --> Apple)

Reply:
yes.  will be in next push